tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -104,6 +105,71 @@ public function userListAllNotifications(string $user):void {
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When user :user deletes all notification
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function userDeletesAllNotification(string $user):void {
+		$this->userListAllNotifications($user);
+		if (isset($this->featureContext->getJsonDecodedResponseBodyContent()->ocs->data)) {
+			$responseBody = $this->featureContext->getJsonDecodedResponseBodyContent()->ocs->data;
+			foreach ($responseBody as $value) {
+				// set notificationId
+				$this->notificationIds[] = $value->notification_id;
+			}
+		}
+		$this->userDeleteNotification($user);
+	}
+
+	/**
+	 * @When user :user deletes the notification related to resource :resource with subject :subject
+	 *
+	 * @param string $user
+	 * @param string $resource
+	 * @param string $subject
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function userDeletesNotificationOfResourceAndSubject(string $user, string $resource, string $subject):void {
+		$this->userListAllNotifications($user);
+		$this->filterResponseByNotificationSubjectAndResource($subject, $resource);
+		$this->userDeleteNotification($user);
+	}
+
+	/**
+	 * deletes notification
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function userDeleteNotification(string $user):void {
+		$this->setUserRecipient($user);
+		$headers = ["accept-language" => $this->settingsContext->getSettingLanguageValue($user)];
+		$payload["ids"] = $this->getNotificationIds();
+		$response = OcsApiHelper::sendRequest(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getActualUsername($user),
+			$this->featureContext->getPasswordForUser($user),
+			'DELETE',
+			$this->notificationEndpointPath,
+			$this->featureContext->getStepLineRef(),
+			\json_encode($payload),
+			2,
+			$headers
+		);
+		$this->featureContext->setResponse($response);
```

Comment:
`setResponse` should be in When step
```suggestion
		return OcsApiHelper::sendRequest(
			$this->featureContext->getBaseUrl(),
			$this->featureContext->getActualUsername($user),
			$this->featureContext->getPasswordForUser($user),
			'DELETE',
			$this->notificationEndpointPath,
			$this->featureContext->getStepLineRef(),
			\json_encode($payload),
			2,
			$headers
		);```

Reply:
I don't see these changes being made so I'm unresolving this.